packages/plugin/webpack/src/Config.ts

```diff
@@ -48,6 +49,20 @@ export interface WebpackPluginEntryPoint {
    * for all entries.
    */
   nodeIntegration?: boolean;
+  /**
+   * Custom options to merge into the configuration passed to `HtmlWebpackPlugin`.
+   */
+  htmlOptions?: Partial<HtmlWebpackPlugin.Options>;
```

Comment:
Instead of `Partial` can this be `Omit` and correctly exclude the keys that we provide

Reply:
yes, good catch, i didn't know about `Omit` :) very cool